addons/l10n_at/data/template/account.account-at.csv

```diff
@@ -61,24 +72,29 @@
 "chart_at_template_0880","Anteile an Personengesellschaften ohne Beteiligungscharakter","0880","False","asset_non_current","l10n_at.account_tag_l10n_at_AAIII5"
 "chart_at_template_0900","Genossenschaftsanteile ohne Beteiligungscharakter","0900","False","asset_non_current","l10n_at.account_tag_l10n_at_AAIII5"
 "chart_at_template_0910","Anteile an Investmentfonds","0910","False","asset_non_current","l10n_at.account_tag_l10n_at_AAIII5"
-"chart_at_template_0980","Geleistete Anzahlungen","0980","False","asset_prepayments","l10n_at.account_tag_l10n_at_AAIII5"
+"chart_at_template_0980","Geleistete Anzahlungen für Finanzanlagen","0980","False","asset_prepayments","l10n_at.account_tag_l10n_at_AAIII5"
 "chart_at_template_0990","Kumulierte Abschreibungen","0990","False","asset_non_current","l10n_at.account_tag_l10n_at_AAIII5"
 "chart_at_template_1600","Handelswaren","1600","False","asset_current","l10n_at.account_tag_l10n_at_ABI3"
-"chart_at_template_1800","Geleistete Anzahlungen","1800","False","asset_prepayments","l10n_at.account_tag_l10n_at_ABI5"
-"chart_at_template_2000","Forderungen von Partnern im Inland ohne eigenes Debitorenkonto","2000","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
-"chart_at_template_2099","Forderungen von Partnern (Point Of Sale)","2099","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
+"chart_at_template_1800","Geleistete Anzahlungen auf Vorräte 20 %","1800","False","asset_prepayments","l10n_at.account_tag_l10n_at_ABI5"
+"chart_at_template_1801","Geleistete Anzahlungen auf Vorräte 10 %","1801","False","asset_prepayments","l10n_at.account_tag_l10n_at_ABI5"
+"chart_at_template_1803","Geleistete Anzahlungen auf Vorräte 0 %","1803","False","asset_prepayments","l10n_at.account_tag_l10n_at_ABI5"
+"chart_at_template_2000","Forderungen aus Lieferungen und Leistungen Inland","2000","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
+"chart_at_template_2099","Forderungen aus Lieferungen und Leistungen Inland (Point Of Sale)","2099","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
 "chart_at_template_2080","Einzelwertberichtigungen zu Forderungen aus Lieferungen und Leistungen Inland","2080","False","asset_current","l10n_at.account_tag_l10n_at_ABII1"
 "chart_at_template_2090","Pauschalwertberichtigungen zu Forderungen aus Lieferungen und Leistungen Inland","2090","False","asset_current","l10n_at.account_tag_l10n_at_ABII1"
-"chart_at_template_2100","Forderungen von Partnern im EU Raum ohne eigenes Debitorenkonto","2100","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
+"chart_at_template_2100","Forderungen aus Lieferungen und Leistungen Währungsunion","2100","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
```

Comment:
"Wirtschafts- und Währungsunion" it would be otherwise it is wrong, but "EU Raum" in short should be ok.

Reply:
EU-Raum einheitlich überall